media/client/ipc/source/IpcClient.cpp

```diff
@@ -30,9 +30,14 @@ IIpcClientAccessor &IIpcClientAccessor::instance()
 
 IIpcClient &IpcClientAccessor::getIpcClient() const
 {
-    static IpcClient ipcClient{ipc::IChannelFactory::createFactory(), ipc::IControllerFactory::createFactory(),
-                               ipc::IBlockingClosureFactory::createFactory()};
-    return ipcClient;
+    static std::unique_ptr<IpcClient> ipcClient;
```

Comment:
Function not thread safe, initialize static variable in one place.
https://laristra.github.io/flecsi/src/developer-guide/patterns/meyers_singleton.html